src/index.ts

```diff
@@ -84,6 +84,10 @@ export default createUnplugin<Options | undefined, false>(
           jsc.target = compilerOptions.target
         }
 
+        if (compilerOptions.useDefineForClassFields === false) {
+          jsc.transform.useDefineForClassFields = false;
```

Comment:
Why not just reference the compiler options here?

Reply:
Do you prefer setting it to `undefined` in case it is not used, rather than not setting it at all?

Reply:
Done